test/signed-log.spec.js

```diff
@@ -5,14 +5,19 @@ const rmrf = require('rimraf')
 const fs = require('fs-extra')
 const Log = require('../src/log')
 const IdentityProvider = require('orbit-db-identity-provider')
+const Keystore = require('orbit-db-keystore')
 
 // Test utils
 const {
   config,
   testAPIs,
   startIpfs,
-  stopIpfs
-} = require('./utils')
+  stopIpfs,
+  implementations
+} = require('orbit-db-test-utils')
+
+const properLevelModule = implementations.filter(i => i.key.indexOf('level') > -1).map(i => i.module)[0]
+const storage = require('orbit-db-storage-adapter')(properLevelModule)
```

Comment:
Could these two lines be moved to test-utils? Then here, one would only need smth like `const { defaultStorage } = require('orbit-db-test-utils'); const storage = defaultStorage`. Doing so would remove the need to specify these two lines in each test.